pub/errors/processor.php

```diff
@@ -257,10 +257,10 @@ public function getViewFileUrl()
         //The url needs to be updated base on Document root path.
         return $this->getBaseUrl() .
         str_replace(
-            str_replace('\\', '/', $this->_indexDir),
+            str_replace('\\', '/', realpath($this->_indexDir)),
```

Comment:
1. Do not use `realpath`, but `\Magento\Framework\Filesystem\DriverInterface::getRealPath($path)`
2. These actions should be performed in constructor, not getter.